UsageCollectorWorkerService/Services/DataHolder/DataHolderService.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text;
+using System.Text.Json;
+using UsageCollectorWorkerService.Models;
+
+namespace UsageCollectorWorkerService.Services.DataHolder;
+
+public class DataHolderService : IDataHolderService
+{
+    public DataHolderService()
+    {
+        Instance = new RootSysResUsageValues()
+        {
+            UsageValues = new List<SysResUsageValues>()
+        };
+    }
+
+    public RootSysResUsageValues Instance { get; set; }
+    
+    public StringContent CreateStringContentForPostRequest(RootSysResUsageValues instance)
```

Comment:
It would be better to move this method to sender service or to some helper/util. Data holder should ideally contain methods insert, get, delete and update (if needed). Also it would be cleaner if Instance would be private and you access the data via these methods.